spyder/defaults/defaults-3.0.0.ini

```diff
@@ -318,7 +318,8 @@ _/debug step into = Ctrl+F11
 _/fullscreen mode = F11
 editor/toggle comment = Ctrl+1
 editor/go to definition = Ctrl+G
-editor/show/hide project explorer = Ctrl+Alt+P
+editor/show/hide project explorer = Ctrl+Alt+J
+_/symbol finder = Ctrl+Alt+P
 _/debug step return = Ctrl+Shift+F11
```

Comment:
Sorry! I thought that file should be modified because a default shortcut was altered and a new one was introduced in this PR